test/kitchen/Berksfile

```diff
@@ -1,5 +1,5 @@
 source 'https://supermarket.chef.io'
-cookbook 'datadog', git: "https://github.com/datadog/chef-datadog.git", branch: "greg/sles"
+cookbook 'datadog', git: "https://github.com/datadog/chef-datadog.git", branch: "arbll/kitchen"
```

Comment:
can you explain what's special about this branch? (in a comment for example)

Reply:
There are some changes on @gmmeyer branch to make SUSE work. Since I had to make a few changes for windows I branched from it. Since https://github.com/DataDog/chef-datadog/pull/520 also fix the issue I had with windows, we could probably pin this to a release once https://github.com/DataDog/chef-datadog/pull/505 is merged

Reply:
cool 👍  can you add an inline comment with: `TODO: pin to a release once https://github.com/DataDog/chef-datadog/pull/520 is released`?